lib/figma.js

```diff
@@ -0,0 +1,83 @@
+const Figma = require('figma-js');
+
+let elements = [];
```

Comment:
This is really needed. I always get nervous when I see these kind of global mutable values inside a module. What makes it so that you can not have a function that just returns elements inside its own scope, by for example composing functions?

Reply:
Something like this? :
```
const data = () => {
  let elements = [];
  return {
    setElements : (element) => {elements = flattenChildren(element)},
    getElements : () => elements
  }
}
```

Reply:
I changed it to this:
```
const data = (() => {
  let elements;
  return {
    setElements : (element) => {elements = flattenChildren(element)},
    getElements : () => elements
  };
})();
```